doc/symfony4.md

```diff
@@ -218,6 +218,17 @@ You will need to create this subscriber class if you use **loggable** , **transl
 behaviors. This listener will set the **locale used** from request and **username** to
 loggable and blameable. So, to finish the setup create **EventSubscriber\DoctrineExtensionSubscriber**
 
+## Register event subscriber for [Symfony Doctrine MongoDB Bundle](https://packagist.org/packages/doctrine/mongodb-odm-bundle)
+
+Because DoctrineExtensions does not implement EventSubscriberInterface from MongoDBBundle, you need to manually tag 
+the listener. Otherwise, the LoggableListener will not be triggered by Doctrine.
```

Comment:
```suggestion
the listeners. Otherwise, they will not be triggered by Doctrine.
```
The idea is that this affects not only `LoggableListener` but all listeners